tb_plugin/test/resnet50_profiler_api.py

```diff
@@ -0,0 +1,83 @@
+import os
```

Comment:
This file is very similar with resnet50_no_next_step.py. Is it better to merge the 2 files into 1 and use cmd argument to control step and no-step?
Please also ask Guoliang's opinion.